RxSwift/Observables/Implementations/Materialize.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Materialize.swift
+//  Rx
+//
+//  Created by sergdort on 08/03/2017.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+
+public protocol EventType {
```

Comment:
I think we need to document `EventType` and all it's properties since it's a part of public API.